docs/cpp/configure-intellisense.md

```diff
@@ -0,0 +1,123 @@
+---
+Order: 13
+Area: cpp
+TOCTitle: Configure IntelliSense
+ContentId: bf494c65-12b4-4506-ab6c-1fad76d7ccf1
+PageTitle: Configure C/C++ IntelliSense
+DateApproved: 8/23/2023
+MetaDescription: Configure Visual Studio Code IntelliSense in the C/C++ Extension
+
+---
+# Configure C/C++ IntelliSense
+
+This article is about configuring the C++ extension to provide C++ specific [IntelliSense](/docs/editor/intellisense.md) suggestions in VS Code. IntelliSense is a helpful tool built into VS Code that provides various code editing features that help you code faster and more efficiently. For example, code completion, parameter info, syntax highlighting, code actions (light bulbs), and member lists are all generated using IntelliSense.
+
+There are three types of sources that can be used for configuration:
+
+- A C/C++ compiler
+- `compile_commands.json` file
+- Another extension (for example, the [Makefile Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.makefile-tools) or [CMake extension](/docs/cpp/CMake-linux.md)).
```

Comment:
I think it should be `for example, the [Makefile Tools](...) or [CMake Tools](..) extension`